src/css/components/header-buttons-mobile.css

```diff
@@ -1,22 +1,34 @@
 .js-overlay-active .mobile-buttons {
   position: fixed;
   top: 1rem;
-  right: 1rem;
+  right: 0;
   z-index: 10;
   left: 0;
-  right: 0;
   background: white;
   height: var(--sp3);
   top: 0;
   padding-right: 1rem;
+  @media (--md) {
```

Comment:
The issue still exists between the viewport widths of 500px and 700px. I haven't tried.. but moving the breakpoint here should fix it. If we change it to 500px, it may be worth adding a custom media query variable.

Reply:
Yeah, I added a lower breakpoint `(--sm)` for it, works fine